### ISSUE TITLE
Honeybadger.Backtrace.format_line fails to format error

### ISSUE THREAD

**Issue by sezaru** [honeybadger-io/honeybadger-elixir#371]
The error occurs when I try to fetch a value from a map using `Map.fetch!`, this will result in the following error:
```elixir
** (FunctionClauseError) no function clause matching in Honeybadger.Backtrace.format_line/1
        (honeybadger 0.16.0) lib/honeybadger/backtrace.ex:27: Honeybadger.Backtrace.format_line({:erlang, :map_get, ["s_2", %{"e_1" => "03:00:00.0Z", "s_1" => "00:00:00.0Z", "t" => 0, "u_a" => "2021-06-03T01:53:41.651335Z", "w" => [false, true, false, true, true, true, true]}], [error_info: %{module: :erl_erts_errors}]})
        (elixir 1.12.1) lib/enum.ex:1553: Enum."-map/2-lists^map/1-0-"/2
        (honeybadger 0.16.0) lib/honeybadger/notice.ex:59: Honeybadger.Notice.new/4
        (honeybadger 0.16.0) lib/honeybadger.ex:292: Honeybadger.notify/2
        (plug 1.11.1) lib/plug/error_handler.ex:98: Plug.ErrorHandler.__catch__/6
        (core 0.1.0) lib/core_web/endpoint.ex:1: CoreWeb.Endpoint.plug_builder_call/2
        (core 0.1.0) lib/core_web/endpoint.ex:1: CoreWeb.Endpoint.call/2
        (phoenix 1.5.9) lib/phoenix/endpoint/cowboy2_handler.ex:65: Phoenix.Endpoint.Cowboy2Handler.init/4
```

**Comment by joshuap** [comment]
This looks like the same error as #369—the fix is currently on master and will be released today or tomorrow. I'll update this issue when it's out!

**Comment by joshuap** [comment]
This should be fixed in 0.16.3. Give it a try and let me know if you continue to have issues. Thanks for reporting. 🏆
